frontend/src/UserInfoView/SignUp.js

```diff
@@ -51,11 +55,11 @@ const SignUp = () => {
       console.log(params);
 
       xhttp.open("POST", api_endpoint, true);
-      xhttp
       xhttp.setRequestHeader("Content-Type", "application/json");
       xhttp.onload = function(){
         // TODO Check if error or success
         console.log(this.response);
+        localStorage.setItem("user", JSON.stringify(this.response.data));
       }
       xhttp.send(JSON.stringify(params));
```

Comment:
Replace this with the `fetch` calls like you wrote in `frontend/src/RequestOptions/user-requests.js`

Reply:
done